root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Framework/Js/common.js

```diff
@@ -0,0 +1,623 @@
+    _div.style.overflow = "auto";
+
+    _div.style.position = "absolute";
+
+    // 1000なら最前面だろうという仕様（ToMost相当が無い）
+    _div.style.zIndex = "1001"; // Maskより前面に出す。
+
+    // 内容を指定
+    _div.innerHTML = "処理中です。しばらくお待ち下さい・・・<hr />";
+    _div.style.backgroundColor = "lightcyan";
+
+    // imgを生成
+    var _img = document.createElement("img");
+
+    _img.src = "/MVC_sample/Framework/Img/loading.gif";
+    _img.style.width = "50px";
+    _img.style.height = "50px";
+    _img.alt = "処理中画像";
+
+    // divにimgを追加
+    _div.appendChild(_img);
+
+    // div → プログレス ダイアログ
+    AjaxProgressDialog = _div
+}
+
+// ---------------------------------------------------------------
+// プログレス ダイアログ表示
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ---------------------------------------------------------------
+
+function Fx_DisplayProgressDialog() {
+    // はじめにタイマをクリアする。
+    clearTimeout(ProgressDialogTimer);
+
+    try {
+        // 表示位置の計算
+        AjaxProgressDialog.style.top = (Fx_getContentsHeight() / 2) //(Fx_getBrowserHeight() / 2)
+            - (AjaxProgressDialog_Height / 2) + "px";
+        AjaxProgressDialog.style.left = (Fx_getBrowserWidth() / 2)
+            - (AjaxProgressDialog_Width / 2) + "px";
+
+        // プログレス ダイアログを表示する。
+        document.body.appendChild(AjaxMask);
+        document.body.appendChild(AjaxProgressDialog);
+
+    } catch (e) {
+        //alert( e );//エラー内容
+    }
+}
+
+//**********************************************************************************
+
+// ------------------------------------------------------------
+//  マスクの表示
+// ------------------------------------------------------------
+
+// Ajax：マスク（div）
+var AjaxMask;
+
+// ---------------------------------------------------------------
+// マスク生成
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ---------------------------------------------------------------
+function Fx_CreateMask() {
+
+    var _div = document.createElement("div");
+
+    _div.style.top = "0px";
+    _div.style.left = "0px";
+    _div.style.height = Fx_getContentsHeight()+"px"; //"100%";では、初期表示画面サイズになってしまう。
+    _div.style.width = Fx_getBrowserWidth() + "px"; //"100%";では、初期表示画面サイズになってしまう。
+    _div.style.position = "absolute";
+
+    // 1000なら最前面だろうという仕様（ToMost相当が無い）
+    _div.style.zIndex = "1000";
+
+    // 分かりやすいように半透明 (0に設定しても良い)
+    _div.style.opacity = 0.5;
+    // IE 8用の透明度の設定(0に設定しても良い)
+    _div.style.filter = "alpha(opacity=50)";
+    // 分かりやすいように着色(若しくは無色)
+    _div.style.backgroundColor = 'gray';
+
+    AjaxMask = _div;
+}
+
+//**********************************************************************************
+
+// ---------------------------------------------------------------
+//  フレームワーク機能（Ajax）
+// ---------------------------------------------------------------
+
+// Ajax：プログレス ダイアログのサイズ（div）
+var AjaxProgressDialog_Width = 300;
+var AjaxProgressDialog_Height = 100;
+
+// Ajax：プログレス中かどうか
```

Comment:
@yusukemaegawa 

I see. But to be honest, it is "too fine".
I will fix this at the same time with the all pointed out fixes.